metrics.go

```diff
@@ -0,0 +1,52 @@
+package ipam
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	prometheusNamespace = "ipam"
+)
+
+var (
+	subnetCounter = prometheus.NewGauge(
+		prometheus.GaugeOpts{
```

Comment:
While total number is interesting, would it be a lot of work to add a number of free subnets ( of course with a standard size like `/24` or `/26` ?
Just an idea as it would be a handy metric to determine and alert on when we are going to run into issues.